### PR TITLE
[docs] release notes and other changes for 2.7.2.0

### PR DESCRIPTION
**Pull request by polarweasel**
@netlify /latest/releases/whats-new/latest-release/

### CODE REVIEW

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 1]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8830](https://github.com/yugabyte/yugabyte-db/issues/8830)] [Platform] Get rid of raw json in AlertReceiver
+* [[8836](https://github.com/yugabyte/yugabyte-db/issues/8836)] [Platform] [UI] Do not reverse the logs
+* [[8841](https://github.com/yugabyte/yugabyte-db/issues/8841)] [Platform] Fetch KMS configs when user has enabled encryption at rest
+* [[8855](https://github.com/yugabyte/yugabyte-db/issues/8855)] [Platform] Restore backup - Selected S3 Default but BackupParams point to NFS
+* [[8858](https://github.com/yugabyte/yugabyte-db/issues/8858)] [Platform] fix playbook for python 3.7 + multiline config value
+* [[8875](https://github.com/yugabyte/yugabyte-db/issues/8875)] [Platform] On failure task runner updates task_info.details.errorString
+* [[8958](https://github.com/yugabyte/yugabyte-db/issues/8958)] [Platform] Fix incorrect sample app command.
+* [[9080](https://github.com/yugabyte/yugabyte-db/issues/9080)] [Platform] Fix certificate value being used for client to server conns
+* [[9109](https://github.com/yugabyte/yugabyte-db/issues/9109)] Deduplicate hostnames in PickResolvedAddress
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:244]: I think this is a minor docdb improvement

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 2]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8481](https://github.com/yugabyte/yugabyte-db/issues/8481)] [Platform] Added check to disable backups if it is in progress. (#8482)
+* [[8487](https://github.com/yugabyte/yugabyte-db/issues/8487)] [Platform] Update index sql version number to prevent conflicts
+* [[8503](https://github.com/yugabyte/yugabyte-db/issues/8503)] [Platform] Fix Add instance modal form to allow for adding instances in isolated region case.
+* [[8525](https://github.com/yugabyte/yugabyte-db/issues/8525)] [Platform] Audit migration failed on AWS portal
+* [[8539](https://github.com/yugabyte/yugabyte-db/issues/8539)] [Platform] [UI] remove toast about auth token expiration
+* [[8540](https://github.com/yugabyte/yugabyte-db/issues/8540)] [Platform] Fix the allowed actions logic
+* [[8541](https://github.com/yugabyte/yugabyte-db/issues/8541)] [Platform] Returns bad request as a response when trying to create a provider with an existing name
+* [[8697](https://github.com/yugabyte/yugabyte-db/issues/8697)] [Platform] Fix client certs not being included when adding custom CA cert.
+* [[8725](https://github.com/yugabyte/yugabyte-db/issues/8725)] Add uid to GetLoggedInUser error message
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:231]: also minor docdb improvement, but test only, so could even discard

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 3]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore for transactional writes
+* [[8543](https://github.com/yugabyte/yugabyte-db/issues/8543)] PITR: Add test for need to increase table version on restore
+* [[8773](https://github.com/yugabyte/yugabyte-db/issues/8773)] PITR: Add DDL log
+* [[8855](https://github.com/yugabyte/yugabyte-db/issues/8855)] Logging improvements for backup restore issue.
+* [[8892](https://github.com/yugabyte/yugabyte-db/issues/8892)] backups: Add timing summary to end of yb_backup
+* [[8894](https://github.com/yugabyte/yugabyte-db/issues/8894)] backups: Allow disabling of checksums in yb_backup
+* [[9046](https://github.com/yugabyte/yugabyte-db/issues/9046)] PITR: Fix crash when using multiple masters
+* [[9157](https://github.com/yugabyte/yugabyte-db/issues/9157)] [DocDB] PITR: Fix operations filters data race
+* [[9171](https://github.com/yugabyte/yugabyte-db/issues/9171)] PITR: Fix restoring snapshot to time before history cutoff
+* [backup] Added a check for 'num_tablets' in CatalogManager::ImportSnapshot.
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:206]: I think this is less PITR and just generic backup related, so could move to normal db improvements

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 4]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8417](https://github.com/yugabyte/yugabyte-db/issues/8417)] PITR: Implement delete_snapshot_schedule
+* [[8418](https://github.com/yugabyte/yugabyte-db/issues/8418)] Add filter to list_snapshot_schedules
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore failover
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore for non transactional writes
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore for transactional writes
+* [[8543](https://github.com/yugabyte/yugabyte-db/issues/8543)] PITR: Add test for need to increase table version on restore
+* [[8773](https://github.com/yugabyte/yugabyte-db/issues/8773)] PITR: Add DDL log
+* [[8855](https://github.com/yugabyte/yugabyte-db/issues/8855)] Logging improvements for backup restore issue.
+* [[8892](https://github.com/yugabyte/yugabyte-db/issues/8892)] backups: Add timing summary to end of yb_backup
+* [[8894](https://github.com/yugabyte/yugabyte-db/issues/8894)] backups: Allow disabling of checksums in yb_backup
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:202]: same, these are generic backup improvements, not strictly PITR related

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 5]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8271](https://github.com/yugabyte/yugabyte-db/issues/8271)] YB-Master should not periodically read the pg_class table if there are no tablespaces.
+* [[8282](https://github.com/yugabyte/yugabyte-db/issues/8282)] Fix commons-lang deps.
+* [[8294](https://github.com/yugabyte/yugabyte-db/issues/8294)] Fix missing conflict when creating row using multiple inserts
+* [[8295](https://github.com/yugabyte/yugabyte-db/issues/8295)] [DocDB] Improved tablet RocksDBs shutdown process
+* [[8323](https://github.com/yugabyte/yugabyte-db/issues/8323)] [YSQL] Add support for USING INDEX TABLESPACE
+* [[8330](https://github.com/yugabyte/yugabyte-db/issues/8330)] [YCQL] Provide capability to skip writing null jsonb attribute in UPDATE stmt
+* [[8347](https://github.com/yugabyte/yugabyte-db/issues/8347)] Wait until executor finish executing async calls on shutdown
+* [[8382](https://github.com/yugabyte/yugabyte-db/issues/8382)] [YSQL] Import Fix mishandling of resjunk columns in ON CONFLICT ... UPDATE tlists.
+* [[8390](https://github.com/yugabyte/yugabyte-db/issues/8390)] Fix NPE Handling for indexed_table_id
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:135]: db bugfix

File: docs/content/latest/releases/whats-new/latest-release.md [hunk 6]
```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
...
+* [[8392](https://github.com/yugabyte/yugabyte-db/issues/8392)] [YSQL] Use negative cache elements to avoid read catalog entries from master
+* [[8473](https://github.com/yugabyte/yugabyte-db/issues/8473)] [YBase] Log the requested memory size in the assertion for allocation failure
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Convert OperationCompletionCallback into std::function
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Simplify MVCCManager to allow aborting only last operation
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Unify ConsensusRound callbacks
+* [[8496](https://github.com/yugabyte/yugabyte-db/issues/8496)] Downgrade gperftools to 2.7
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-release.md:141]: bugfix -- I think we called this out in stable versions, but for latest, prolly not worth it

**Review by ymahajan — APPROVED** [review]

**Review by m-iancu — APPROVED** [review]

**Pull request merged**